tools/wasm2c/base.c

```diff
@@ -10,10 +10,16 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include <sys/stat.h>
-#include <sys/types.h>
 #include <time.h>
-#include <unistd.h>
+
+// ssize_t detection: usually stdint provides it, but not on windows apparently
+#ifndef ssize_t
```

Comment:
I don't think `ifndef` is what you want here as that doesn't detect typedefs.  

I also don't believe that windows headers don't define this :)

Reply:
I'm open to other ideas, but this is what I found when searching, and it seems to work... (at least to compile, i haven't gotten the code to link and run yet).

Reply:
At the very least I don't see how `ifdef` can work here because size_t is not a macro.. its at typedef, no?

Reply:
Yeah, I don't get it either, I was just copying code... I pushed another version without that now.